Branch2018/test1/Bill.cs

```diff
@@ -0,0 +1,27 @@
+namespace Object
+{
+    public class Bill
+    {
+        public Bill()
+        {
+        }
+
+        public int BillId { get; set; }
+
+        public int SellerId { get; set; }
+
+        public int ProductId { get; set; }
+
+        public int CodeBill { get; set; }
+
+        public int NumberRUC { get; set; }
```

Comment:
the right way will be RucNumber, but Ruc is not a known name around the world, in that case would "TIN" or something more but it's not very important  for the test.